tools/harness-thci/OpenThread.py

```diff
@@ -67,9 +67,15 @@
 from commissioner_impl import OTCommissioner
 
 # Replace by the actual version string for the vendor's reference device
-OT11_VERSION = 'ot11_version'
-OT12_VERSION = 'ot12_version'
-OT13_VERSION = 'ot13_version'
+OT11_VERSION = 'OPENTHREAD'
+OT12_VERSION = 'OPENTHREAD'
+OT13_VERSION = 'NotSupported'
+
+# Supported device capabilites in this THCI implementation
+OT11_CAPBS = DevCapb.V1_1
+OT12_CAPBS = (DevCapb.L_AIO | DevCapb.C_FFD | DevCapb.C_RFD)
+OT12BR_CAPBS = (DevCapb.C_BBR | DevCapb.C_Host | DevCapb.C_Comm)
+OT13_CAPBS = DevCapb.NotSpecified
```

Comment:
Looks like a workaround to use `DevCapb.NotSpecified` for OT13?

Should we also define OT13 capabilities?

Reply:
OT13 capabilities are not part of any Harness release yet, this is just here as a reminder for adding them when available.